src/go/rpk/go.mod

```diff
@@ -37,7 +37,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/prometheus/client_model v0.6.1
 	github.com/prometheus/common v0.59.1
-	github.com/redpanda-data/common-go/rpadmin v0.1.6
+	github.com/redpanda-data/common-go/rpadmin v0.1.7
```

Comment:
@r-vasquez, should this be `v0.1.6`? v24.2.x still uses `v0.1.6`, https://github.com/redpanda-data/redpanda/blob/v24.2.x/src/go/rpk/go.mod#L40

Reply:
Backporting https://github.com/redpanda-data/redpanda/commit/1f1e0cd779d794057621b8bf95a444d87c785d07, which bumped the rpadmin version,  via https://github.com/redpanda-data/redpanda/pull/23689

Reply:
It should be fine as your PR also bumped the rpadmin version to v0.1.7